crates/sui-framework/packages/bridge/tests/bridge_setup.move

```diff
@@ -0,0 +1,695 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[test_only]
+module bridge::bridge_setup {
+    use bridge::bridge::{
+        assert_not_paused, assert_paused, create_bridge_for_testing,
+        test_execute_add_tokens_on_sui, test_execute_emergency_op, test_init_bridge_committee,
+        test_load_inner_mut, Bridge,
+    };
+    use bridge::btc::{Self, BTC};
+    use bridge::eth::{Self, ETH};
+    use bridge::message::{
+        Self, create_add_tokens_on_sui_message, create_blocklist_message, emergency_op_pause,
+        emergency_op_unpause,
+    };
+    use bridge::test_token::TEST_TOKEN;
+    use bridge::usdc::{Self, USDC};
+    use bridge::usdt::{Self, USDT};
+    use std::type_name;
+    use sui::coin::{Self, Coin, CoinMetadata, TreasuryCap};
+    use sui::hex;
+    use sui::package::UpgradeCap;
+    use sui::test_scenario::{Self, Scenario};
+    use sui::test_utils::destroy;
+    use sui_system::{
+        governance_test_utils::{
+            advance_epoch_with_reward_amounts,
+            create_sui_system_state_for_testing,
+            create_validator_for_testing,
+        },
+        sui_system::{
+            validator_voting_powers_for_testing,
+            SuiSystemState,
+        },
+    };
+
+    //
+    // Token IDs
+    //
+    const BTC_ID: u8 = 1;
+    const ETH_ID: u8 = 2;
+    const USDC_ID: u8 = 3;
+    const USDT_ID: u8 = 4;
+    // const TEST_TOKEN_ID: u8 = 5;
```

Comment:
remove?

Reply:
not really, I am going to add functions that return those and it's not worth to remove it for now